### ISSUE TITLE
[Request] [Not Important] Asymmetric 

### ISSUE THREAD

**Issue by Zoryn4163** [sinkillerj/ProjectE#586]
When doing some testing I found that items placed on the Dark Matter Pedestal that have an effect are asymmetric. The radius is 10x9x10 (X, Y, Z), which isn't natural-looking for a one-block-wide block. Just requesting that this be changed at some point as I believe it's an easy thing to change.
http://i.imgur.com/UahAf7d.jpg
![image](https://cloud.githubusercontent.com/assets/11305496/6884232/53d1ac4a-d5b4-11e4-9e01-361da34db16e.png)
Basically the pink blocks should be 1x1 instead of 2x2, but the effect applies to an even area instead of an odd area horizontally.

**Comment by Nova001Sol** [comment]
NOT IMPORTANT?! I disagree, this is a matter of great importance. It will need to be fixed posthaste.

**Comment by MaPePeR** [comment]
i think its not that important, but it has a good importance/effort to fix ratio.

**Comment by sinkillerj** [comment]
OCPD is very important MaPePeR

**Comment by Zoryn4163** [comment]
It's simply a request to be fixed. If you have more important features to be working on this can sit in the back-burner for as long as you like. I tried looking into your code but don't know enough about it to be able to have found a place to change, at least not in raw numbers.

**Comment by williewillus** [comment]
I was the one who wrote it so I should be able to get to it really quickly...but I don't really get the problem from your post. Is it visual? Do you mean the actual field the pedestal applies its effects within?

**Comment by Zoryn4163** [comment]
Thank you and yes that's what I meant. The effect was applying in a 10x9x10 area though. so I'm not sure where you got 10x10x10 from...
